src/ol_infrastructure/substructure/vault/pki/__main__.py

```diff
@@ -19,70 +17,35 @@
 stack_info = parse_stack()
 pki_intermediate_export = {}
 
-# The files should be generated by the offline Root CA and copied locally prior to
-# running this. More information here - https://docs.odl.mit.edu/en/platform/security/vault-pki  # noqa: E501
-try:
-    int_ca_pem_bundle_file = open("/tmp/int_ca_bundle_file.pem").read()
-    int_ca_cert_file = open("/tmp/int_ca_cert_file.pem").read()
-    pki_intermediate_ca_config = OLVaultPKIIntermediateCABackendConfig(
-        max_ttl=TWELVE_MONTHS,
-        default_ttl=TWELVE_MONTHS,
-        intermediate_ca_pem_bundle=int_ca_pem_bundle_file,
-        intermediate_ca_cert=int_ca_cert_file,
-    )
-except OSError:
-    raise OSError("Could not find or read cert files")  # noqa: B904
+acmpca_stack = StackReference("infrastructure.aws.private_ca")
+root_ca_arn = acmpca_stack.require_output("root_ca").apply(lambda rca: f"{rca['arn']}")
 
+if Config("vault_server").get("env_namespace"):
```

Comment:
This shouldn't need the conditional. We can just run the provider setup unconditionally. That pattern was a temporary accommodation while we migrated to the automatic provider.